.jsdoc.json

```diff
@@ -35,6 +35,7 @@
   },
   "plugins": ["plugins/markdown"],
   "markdown": {
-    "tags": ["example"]
+    "tags": ["example"],
+    "idInHeadings": true
```

Comment:
The way that marked/jsdoc add anchor links isn't quite compatible with GFM, so, it warranted the changes in `fix-api-docs.js` below.